src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/MappaDetails.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation
+
+class MappaDetails(
```

Comment:
There seem to be a lot of new classes for things that I don't believe we ever use. Do we need these?
I may be mistaken, but I believe you can leave out references to fields you don't need from API types, and it will still recognise the object as long as there's overlap with the existing fields.

In other words, you should be able to remove fields we don't use from `OffenderDetail.kt`, which would mean that we wouldn't need all of these new classes

Reply:
Yes, deleted unused models.